frontend/src/routes/Home/Search/SearchPage.tsx

```diff
@@ -131,19 +132,20 @@ function RenderSearchBar(props: {
 
   const { searchCompleteValue, searchCompleteQuery } = useMemo(() => {
     const value = getSearchCompleteString(currentSearch)
-    const query = convertStringToQuery(currentSearch)
+    const query = convertStringToQuery(currentSearch, searchQueryLimit)
     query.filters = query.filters.filter((filter) => {
       return filter.property !== value
     })
     return { searchCompleteValue: value, searchCompleteQuery: query }
-  }, [currentSearch])
+  }, [currentSearch, searchQueryLimit])
 
   const searchCompleteResults = useSearchCompleteQuery({
     skip: !currentSearch.endsWith(':') && !operators.some((operator: string) => currentSearch.endsWith(operator)),
     client: process.env.NODE_ENV === 'test' ? undefined : searchClient,
     variables: {
       property: searchCompleteValue,
       query: searchCompleteQuery,
+      limit: 10000,
```

Comment:
Should read from env. `useSearchQueryLimit()`

Reply:
updated.